nomad/structs/diff.go

```diff
@@ -1559,6 +1564,103 @@ Loop:
 	return diff
 }
 
+// volumeDiffs returns the diff of a group's volume requests. If contextual
+// diff is enabled, all fields will be returned, even if no diff occurred.
+func volumeDiffs(old, new map[string]*VolumeRequest, contextual bool) []*ObjectDiff {
```

Comment:
Kinda wish we wouldn't shadow `new`

Reply:
Oof... good call. I've fixed it in this PR but in the interest of not making a huge diff of unrelated style changes I'm not going to fix it where we're doing it everywhere else in `diff.go` in this PR